src/main/java/uk/gov/cshr/civilservant/service/OrganisationalUnitService.java

```diff
@@ -164,11 +166,23 @@ public AgencyTokenResponseDto getAgencyToken(Long organisationalUnitId) throws C
 
     @Cacheable("organisationalUnitsTree")
     public List<OrganisationalUnit> getOrgTree() {
-        return this.getParents();
+        List<OrganisationalUnit> listOrg = this.getParents();
+        sortOrganisationList(listOrg);
+        return listOrg;
     }
 
     @Cacheable("organisationalUnitsFlat")
     public List<OrganisationalUnitDto> getFlatOrg() {
         return this.getListSortedByValue();
     }
+
+    private void sortOrganisationList(List<OrganisationalUnit> list) {
+        list.forEach(org ->
+            {
+                if(org.hasChildren())
```

Comment:
Based on the indentation of the next two lines, it seems that these are part of the if true block but the opening and closing curly braces are missing for the if true block so only the first line after the if will be included in the if true block. Could you please add the curly braces correctly for if true block.